test/files/pos/CharSequence-implicits.scala

```diff
@@ -0,0 +1,4 @@
+object Test extends App {
+  Array('a', 'b'): CharSequence
+  IndexedSeq('a', 'b'): CharSequence
+}
```

Comment:
Adriaan was also surprised by this; I will investigate separately, but not right away. (In the meantime, it doesn't affect merging this.)